### ISSUE TITLE
Asses accessibility of Checkbox

### ISSUE THREAD

**Issue by tobiasljungstrom** [SKF/ferris#228]
Asses the accessibility of the Checkbox component and mark it in the component table.

The component should meet the following criteria (if applicable):

- [x] Tab-able with a clear focus state
- [x] Screen reader should be able to traverse it
- [x] Custom control should work with screen reader
- [x] Focus should be directed to any new items that pop up on the screen
- [x] Color and contrast should be good
